plugins/snapcraft_travis/snapcraft_travis.py

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/python3
+# -*- Mode:Python; indent-tabs-mode:nil; tab-width:4 -*-
+#
+# Copyright (C) 2017 Canonical Ltd
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License version 3 as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+import json
+import requests
+
+import errbot
+import git
+import github
+
+
+class SnapcraftTravis(errbot.BotPlugin):
+    """Handle Travis requests with the Travis API."""
+
+    def activate(self):
+        super().activate()
+
+    @errbot.arg_botcmd('pull_request_number', type=int)
+    def travis_snapurl(self, message, pull_request_number):
+        try:
+            build_id = self._get_build_id(pull_request_number)
+        except errbot.ValidationException:
+            return 'Could not find build with pull request #%i' % pull_request_number
```

Comment:
The new way to format strings in python is using the format method. Also, because of some bot fights that happen in the IRC channel, we have to prefix the PR number with `snapcraft`.

Please change this to:

'Could not find build with pull request snapcraft#{}'.format(pull_request_number)